src/Builder/DatagridBuilder.php

```diff
@@ -135,7 +135,6 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
         //    $fieldDescription->setOption('parent_association_mappings', $fieldDescription->getOption('parent_association_mappings', $fieldDescription->getParentAssociationMappings()));
         //}
 
-        $fieldDescription->setOption('code', $fieldDescription->getOption('code', $fieldDescription->getName()));
```

Comment:
why this change?

Reply:
Cause now, passing a non working `code` throw an exception:
https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Admin/BaseFieldDescription.php#L404-L430